### PR TITLE
Update to Go 1.19

### PR DESCRIPTION
**Pull request by clobrano**
Signed-off-by: Carlo Lobrano <c.lobrano@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -140,23 +140,23 @@ undeploy:
...
-define go-get-tool
+define go-install-tool
 @[ -f $(1) ] || { \
 set -e ;\
 TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 go mod init tmp ;\
 echo "Downloading $(2)" ;\
-GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
```
Comment (slintes) [inline Makefile:159]: Somehow the OCP Go image, which is used in CI (see `.ci-operator.yaml`), has a fixed `-mod=vendor` flag, which doesn't work when installing tools with temporary modules.
You need to add GOFLAGS='':
`GOBIN=$(PROJECT_DIR)/bin GOFLAGS='' go install $(2) ;\`

Comment (clobrano) [reply Makefile:159]: I see now, thank you :bow:

**Comment by mshitrit** [comment]
/lgtm
/hold
Since there are no e2e tests yet I suggest (if you haven't already) verifying manually that the operator works as expected.

**Comment by clobrano** [comment]
>  Since there are no e2e tests yet I suggest (if you haven't already) verifying manually that the operator works as expected.

I did tested it (before and after this PR, to have a reference), and it is working as expected

**Review by slintes — CHANGES_REQUESTED** [review]
when I run `make generate manifests test` I see some uncommitted changes, please have a look

**Review by slintes — APPROVED** [review]
/lgtm
/approve

**Comment by slintes** [comment]
oh, still WIP
feell free to remove when done 🙂 

/hold

**Comment by slintes** [comment]
nevermind, WIP will not be merged anyway 😆 

/hold cancel

**Comment by clobrano** [comment]
LOL, sorry for the noise :smile: 
It is ready again

**Pull request merged**